modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i

```diff
@@ -43,262 +43,232 @@
 []
 
 [Variables]
-  [./disp_x]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-  [./disp_y]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-  [./disp_z]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-  [./rot_x]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-  [./rot_y]
-    order = FIRST
-    family = LAGRANGE
-  [../]
+  [disp_x]
+  []
+  [disp_y]
+  []
```

Comment:
Cryptic use of defaults.

Reply:
I removed those as they are IMO the most basic thing and clutter up the input considerably. But I can undo this change. I have to fix the SQA error anyways.

Reply:
I am fine with it.

Reply:
Oh. well, i rolled it back already. Maybe best to stick to a small change set anyways. I now see that you actually approved the PR with the comment above. This is not visible in the github webinterface at all (only saw that in my email). The comment sounded more like you want this changed.

Reply:
Among the many lines of input file needed to set up that problem, one main piece of information should be the element type. Not sure if I agree that the default option here is the most basic thing. It's one common finite element option that happens to be default. 

Somehow the approval got stale. Not sure how.